### DISCUSSION TITLE
Best way to update a Formula in homebrew-core? libcsv is old and lives on GitHub now

### DISCUSSION THREAD

**Discussion by ghenry** [Homebrew/discussions#2940] [Tap maintenance and Homebrew development] [answered]
I'd like to upgrade:

https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/libcsv.rb

as it all lives here now:

https://github.com/rgamble/libcsv

note on SF.

Thanks.

**Comment by SMillerDev** [comment, marked as answer]
https://docs.brew.sh/Updating-Software-in-Homebrew

**Reply by ghenry** [reply]
Thanks. Will do.
